pages/content.html

```diff
@@ -0,0 +1,161 @@
+---
+permalink: /content-style/
+layout: styleguide
+title: Content Style
+slug: content-style
+---
+
+<h1>Content Style</h1>
+
+<h2>Writing for government digital services</h2>
+
+<p>here are many agencies and people contributing content to government digital services, and this style guide is intended to help us make all government information readable and understandable. This style guide borrows from the <a href="https://github.com/GSA/data.gov-styleguide">Data.gov's Style Guide</a>, which is based on <a href="https://www.gov.uk/design-principles/style-guide">UK's Style Guide</a> and will be updated as needs evolve. <strong>We welcome suggestions and feedback!</strong></p>
```

Comment:
Under "We welcome suggestions and feedback!", it might be nice to make that link to https://github.com/18F/govt-wide-patternlibrary/issues/new. 

@ErieMeyer :+1: or :-1: ?